app/src/organisms/Desktop/Camera/CameraControls/PreviewSettings/hooks/usePreviewImage.ts

```diff
@@ -1,18 +1,33 @@
 import { useState } from 'react'
 
+import { useCapturePreviewImage } from '@opentrons/react-api-client'
+
+import type { CameraImageSettings } from '@opentrons/api-client'
+
 export interface UsePreviewImageResult {
   imgPath: string | undefined
   isLoading: boolean
   takePhoto: () => void
 }
 
-export function usePreviewImage(): UsePreviewImageResult {
-  const [isLoading, setIsLoading] = useState(false)
+export function usePreviewImage(
+  settings: CameraImageSettings
+): UsePreviewImageResult {
   const [imgPath, setImgPath] = useState<string | undefined>(undefined)
+  const [isLoading, setIsLoading] = useState(false)
 
-  const takePhoto = (): void => {
-    setIsLoading(true)
+  const { mutate: captureImage } = useCapturePreviewImage({
```

Comment:
```suggestion
  const { createCapturePreviewImage } = useCapturePreviewImage({
```
That works, but it's more internally consistent with how we utilize hooks to destructure our custom mutation, which has the added benefit or letting us change the custom mutation and not have to update all the call sites.